src/model/history/add-object-change.ts

```diff
@@ -21,9 +21,8 @@ export class AddObjectChange extends Change {
     return set(cloneManuscript(manuscript), this.path, [...originalSection, this.object]);
   }
 
-  public get isEmpty() { 
-    console.log('isEmpty not implimented for AddObjectChange')
-    return false; 
+  public get isEmpty() {
+    return false;
```

Comment:
I know this was probably removed by a linter but we should probably still flag that this functions not been implemented properly (If you could correct my miss-spelling of implemented along the way that'd be great)

Reply:
We could potentially change these to a `console.warn` to make it clear they're warnings

Reply:
both AddObject and DeleteObject have this implementation in the client as well.